neural_lam/models/ar_model.py

```diff
@@ -65,10 +65,12 @@ def __init__(
                 da_state_stats.state_std.values, dtype=torch.float32
             ),
             "diff_mean": torch.tensor(
```

Comment:
Just an additional comment, to be extra clear about this
```suggestion
            # Note that the one-step-diff stats (diff_mean and diff_std) are for differences 
            # computed on standardized data
            "diff_mean": torch.tensor(
```